dittybopper/deploy.sh

```diff
@@ -108,8 +108,9 @@ fi
 #FIXME: This is OCP-Specific; needs updating to support k8s
 echo ""
 echo -e "\033[32mGetting environment vars...\033[0m"
-export PROMETHEUS_URL=$($k8s_cmd get secrets -n openshift-monitoring grafana-datasources -o go-template='{{index .data "prometheus.yaml" | base64decode }}' | jq '.datasources[0].url')
-export PROMETHEUS_PASSWORD=$($k8s_cmd get secrets -n openshift-monitoring grafana-datasources -o go-template='{{index .data "prometheus.yaml"| base64decode }}' | jq 'if .datasources[0].basicAuthPassword != null then .datasources[0].basicAuthPassword else .datasources[0].secureJsonData.basicAuthPassword end')
+export GRAFANA_SECRET=$($k8s_cmd get secrets -n openshift-monitoring -l app.kubernetes.io/component=grafana | grep datasource | awk '{print $1}')
```

Comment:
Small nit, we don't need to export GRAFANA_SECRET

Reply:
you are right, I will correct that.